autoload/neomake/makers/ft/typescript.vim

```diff
@@ -1,7 +1,7 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#typescript#EnabledMakers() abort
-    return ['tsc', 'tslint']
+    return ['tsc', 'tslint', 'eslint']
```

Comment:
How does it behave if you have eslint installed, but not https://github.com/typescript-eslint/typescript-eslint?
(I wonder if it should be enabled by default).

Reply:
Good question, it would fail end probably cause an error. I removed eslint from the default makers.